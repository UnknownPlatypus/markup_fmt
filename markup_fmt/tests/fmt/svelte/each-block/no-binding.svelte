{#each { length: 10 }}
  <p>fails</p>
{/each}

{#each { length: 10 }, index}
  <p>fails</p>
{/each}
