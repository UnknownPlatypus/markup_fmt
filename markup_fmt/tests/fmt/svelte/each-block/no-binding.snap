---
source: markup_fmt/tests/fmt.rs
---
{#each { length: 10 }}
  <p>fails</p>
{/each}

{#each { length: 10 }, index}
  <p>fails</p>
{/each}
